packages/manager/modules/carrier-sip/src/carrier-sip.service.js

```diff
@@ -0,0 +1,72 @@
+import map from 'lodash/map';
+
+import VoipCarrierSipLine from './VoipCarrierSipLine.class';
+
+export default class CarrierSipService {
+  /* @ngInject */
+  constructor(OvhApiTelephony) {
+    this.OvhApiTelephony = OvhApiTelephony;
+  }
+
+  fetchAll(billingAccount) {
+    return this.OvhApiTelephony.CarrierSip().Iceberg()
+      .query()
+      .expand('CachedObjectList-Cursor')
+      .execute({ billingAccount })
+      .$promise
+      .then(({ data: carrierSipLines }) => (map(
+        carrierSipLines,
+        carrierSipLine => new VoipCarrierSipLine({
+          billingAccount,
+          ...carrierSipLine,
+        }),
+      )));
+  }
+
+  getCarrierSip(billingAccount, serviceName) {
+    return this.OvhApiTelephony.CarrierSip().v6().get({
+      billingAccount,
+      serviceName,
+    }).$promise.then(carrierSip => new VoipCarrierSipLine({
+      ...carrierSip,
+      billingAccount,
+    }));
+  }
+
+  getSettings(billingAccount, serviceName) {
+    return this.OvhApiTelephony.CarrierSip().Settings().v6().query({
+      billingAccount,
+      serviceName,
+    }).$promise;
+  }
+
+  getServiceInfos(serviceName) {
+    return this.OvhApiTelephony.Lines().v6().getServiceInfos({
+      serviceName,
+    }).$promise;
+  }
+
+  getEndpoints(billingAccount, serviceName) {
+    return this.OvhApiTelephony
+      .CarrierSip()
+      .Endpoints()
+      .v6()
+      .query({
+        billingAccount,
+        serviceName,
+      })
+      .$promise
+      .then(endpointsIds => Promise.all(map(
```

Comment:
I think we should avoid `Promise` as long as we use AngularJS, even though it works here :)

Reply:
We should talk about it ;) 
for now I'll replace with a `$q`